Rocket.Chat/Views/Chat/RCTextView.swift

```diff
@@ -135,8 +141,6 @@ class HighlightLayoutManager: NSLayoutManager {
         super.layoutSubviews()
 
         textView.frame = bounds
-
-        updateCustomEmojiViews()
```

Comment:
By calling `updateCustomEmojiViews()` on `layoutSubviews` we were calling it A LOT of unnecessary times.